core/templates/dev/head/components/question-directives/question-player/question-player.constants.ts

```diff
@@ -27,18 +30,12 @@ angular.module('oppia').constant(
     // color shallow green
     PASSED_COLOR_OUTER: 'rgb(143, 217, 209)'
   }
-);
-
-angular.module('oppia').constant('HASH_PARAM', 'question-player-result=');
-
-angular.module('oppia').constant('MAX_SCORE_PER_QUESTION', 1.0);
 
-angular.module('oppia').constant(
-  'QUESTION_PLAYER_MODE', {
+  public static QUESTION_PLAYER_MODE = {
     PASS_FAIL_MODE: 'PASS_FAIL'
   }
-);
 
-angular.module('oppia').constant('VIEW_HINT_PENALTY', 0.1);
+  public static VIEW_HINT_PENALTY = 0.1
```

Comment:
Are semicolons needed at the ends of these lines? It seems inconsistent at the moment -- some have them and some don't. Let's be consistent.

(Probably prefer the version that has semicolons, since that's standard in JS, though I'm a little less famliiar with JS classes. @kevinlee12, any thoughts?)

Reply:
Semicolons aren't needed, though I can enable that eslint rule.

Reply:
Yep, let's stipulate a style, and have eslint check it, so that everything stays consistent.